apps/web/src/modules/transaction/pages/casting/components/PendingCastingCard/index.tsx

```diff
@@ -27,12 +26,10 @@ const PendingCastingCard = (props: PendingCastingCardProps) => {
 
   return (
     <CardContainer>
-      <JobCardHeader
+      <BaseHeader
         title={castingName}
         companyName={companyName}
         jobCastingImageUrl={''}
```

Comment:
Note: Currently, API does not send casting profile image url

Reply:
ตอนแรกเข้าใจว่า PendingCasting คือการ์ดของ casting ที่ admin ยังไม่ approve เลยคิดว่าไม่มี imageUrl คือถูกแล้ว...

Reply:
อันนี้มันคือหน้า Pending Transaction ของ Casting อะ